src/cwe_checker_lib/src/analysis/stack_alignment_substitution/mod.rs

```diff
@@ -86,6 +88,59 @@ fn journal_sp_value(
     }
 }
 
+/// Returns the tid of the target of the first Jmp::Branch of the provided block.
+fn get_first_branch_tid(blk: &Term<Blk>) -> Option<&Tid> {
+    for jmp in &blk.term.jmps {
```

Comment:
I think this should be
```rust
if let Some(jmp) = blk.term.jmps.get(0) {
```
The `for`-loop would otherwise also catch one branch of conditional jumps, which is not what we want here.